ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -81,6 +81,12 @@
           ROWOFFSET_NAME
       );
 
+  private static final Set<ColumnName> DISALLOWED_FOR_PULL_QUERIES =
```

Comment:
This is now a fourth thing that anyone adding a new pseudocolumn needs to be reason about. Can we please make the changes described in https://github.com/confluentinc/ksql/pull/8121#discussion_r707600810?

Reply:
Sorry about that, haven't read that code review yet. I'll make a `PseudoColumn` class.